tests/devices/qubit/test_simulate.py

```diff
@@ -1178,3 +1184,228 @@ def test_qinfo_tf(self):
 
         grad5 = grad_tape.jacobian(results[5], phi)
         assert qml.math.allclose(grad5, expected_grads[5])
+
+
+class TestMidMeasurements:
+    """Tests for simulating scripts with mid-circuit measurements using the ``simulate_tree_mcm``."""
+
+    @pytest.mark.parametrize("val", [0, 1])
+    def test_basic_mid_meas_circuit(self, val):
```

Comment:
```suggestion
    @pytest.mark.unit
    @pytest.mark.parametrize("val", [0, 1])
    def test_basic_mid_meas_circuit(self, val):
```
Can you add the test level markers to the rest of the tests as well?